### DISCUSSION TITLE
Error when creating a bool column

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#2025] [Questions] [answered]
Hi 👋🏻 

When I'm trying to create a boolean column for my users table, it throws me an error "Error for input at Type: Field cannot be empty"

![See the bug here: https://i.imgur.com/anrbtFo.gif](https://i.imgur.com/anrbtFo.gif)

Does anyone know how to fix this?

**Comment by inian** [comment, marked as answer]
Hi @Olyno, confirmed this is a bug and we are fixing this.
